internal/operator-controller/applier/helm_test.go

```diff
@@ -163,6 +155,33 @@ spec:
 	testCE            = &ocv1.ClusterExtension{}
 	testObjectLabels  = map[string]string{"object": "label"}
 	testStorageLabels = map[string]string{"storage": "label"}
+	errPreAuth        = errors.New("problem running preauthorization")
+	missingRBAC       = []authorization.ScopedPolicyRules{
+		{
+			Namespace: "",
+			MissingRules: []rbacv1.PolicyRule{
+				{
+					Verbs:           []string{"list", "watch"},
+					APIGroups:       []string{""},
+					Resources:       []string{"services"},
+					ResourceNames:   []string(nil),
+					NonResourceURLs: []string(nil)},
+			},
+		},
+		{
+			Namespace: "test-namespace",
+			MissingRules: []rbacv1.PolicyRule{
+				{
+					Verbs:     []string{"create"},
+					APIGroups: []string{"*"},
+					Resources: []string{"certificates"}},
+			},
+		},
+	}
+
+	errMissingRBAC = "pre-authorization failed: service account requires the following permissions to manage cluster extension:\n" +
+		"  Namespace:\"\" APIGroups:[] Resources:[services] Verbs:[list,watch]\n" +
+		"  Namespace:\"test-namespace\" APIGroups:[*] Resources:[certificates] Verbs:[create]\n"
```

Comment:
I would suggest using backticks to format multi-line strings and/or strings containing quotation marks:
```suggestion
	errMissingRBAC = `pre-authorization failed: service account requires the following permissions to manage cluster extension:
  Namespace:"" APIGroups:[] Resources:[services] Verbs:[list,watch]
  Namespace:"test-namespace" APIGroups:[*] Resources:[certificates] Verbs:[create]
`
```